llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1447,6 +1447,152 @@ LogicalResult TopkOp::getResultTilePosition(
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// PackOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult PackOp::verify() {
```

Comment:
I am assuming this is going to be flushed out once we have the op nailed down?

Reply:
Yes, just added some basic checks now. But still not complete.